orchagent/portsorch.cpp

```diff
@@ -553,7 +553,9 @@ bool PortsOrch::bindAclTable(sai_object_id_t id, sai_object_id_t table_oid, sai_
             port.m_egress_acl_table_group_id = groupOid;
         }
 
-        gCrmOrch->incCrmAclUsedCounter(CrmResourceType::CRM_ACL_GROUP, (sai_acl_stage_t) group_attr.value.s32, SAI_ACL_BIND_POINT_TYPE_PORT);
+        sai_acl_stage_t acl_stage = ingress ? SAI_ACL_STAGE_INGRESS : SAI_ACL_STAGE_EGRESS;
```

Comment:
This was a bug that was fixed and merged recently. Can you check the latest code?

Reply:
I have checked the latest code on master branch, but did not see the fix code. Could you please tell me in which commit it merged? thank you.

Reply:
https://github.com/Azure/sonic-swss/pull/496